README.md

```diff
@@ -23,7 +23,7 @@ docker pull dhiway/cord:develop
 ```
 
 
-### Create an account and ask for membership.
+### Create an account and ask for membership ( [Account.md](https://github.com/dhiway/confidex/blob/main/Account.md) ).
```

Comment:
the ideal way is having the relative path inside the repo. That will enable it to rename repo without any problem with links.

Reply:
am thinking what'll happen if the files are individually copied over to some other locations

Reply:
lets keep the link. If the files are renamed or moved to other locations, then that needs to be handled with proper link change here.

Reply:
I mean - if the users copy it over to some other locations, then these links will not work